### PR TITLE
week 2 walkthrough

### PR DESCRIPTION
**Pull request by nitishbharti235**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("Hello Nitish. \t Just wanted to tell you, Ur Special!!");
```
Comment (pitankar) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: nit: `Ur` is intentional? or it had to be `You are`?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -34,3 +34,17 @@ function addRandomQuote() {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+function randomMessage(){
+  console.log('Inside randomMessage function.. ');
+  // const msgContainer =  document.getElementById('msg-container');
+  // msgContainer.innerText = "Ur Special!!";
```
Comment (pitankar) [inline portfolio/src/main/webapp/script.js:41]: Should these be removed?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (nitishbharti235) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: It is intentional..
there is a popular youtube creater `coolman coffeedan`.. he used to say that.. so I included it :sweat_smile:

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (nitishbharti235) [reply portfolio/src/main/webapp/script.js:41]: I'm keeping this.. to remember the old way of doing it.. :sweat_smile:

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (pitankar) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: lol! that's fine then!

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (pitankar) [reply portfolio/src/main/webapp/script.js]: Do you want to add a line on top of these (on 39) stating the same. So others looking at this in later patches don't confuse this with dead/unused code and understand the reasons for it better.

Comment (nitishbharti235) [reply portfolio/src/main/webapp/script.js]: there are lot of changing in the tutorials.. I'm removing the older way..
new ways are much better and cooler.

**Pull request merged**